tests_against_server/test_loops.py

```diff
@@ -95,20 +90,8 @@ def IQ_imbalance(g, phi):
     }
 
 
-def simulate_program_and_return(config, prog, duration=50000):
-
+def simulate_program_and_return(config, prog):
     qmm = QuantumMachinesManager()

```

Comment:
How can you run this without supplying any arguments to the constructor? Are you trying to use the conftest? If so, you can just put `qmm` as the first argument to the test function, i.e.,

`def simulate_program_and_return(**qmm,** config, prog):`

Reply:
Ok that's fine. But on line 94, you leave the inputs empty. When I run that on my machine I get this:

```python
>>> from qm import QuantumMachinesManager
2024-04-10 22:24:00,708 - qm - INFO     - Starting session: 7c33597d-ada4-4350-8bd5-35ff41f888b1
>>> qmm = QuantumMachinesManager()
2024-04-10 22:24:06,152 - qm - ERROR    - Failed to detect a QuantumMachines server. Tried connecting to :80.
...
```

Reply:
Yes I leave it empty once I am done so that the whole world doesn't see our IP address and port